stats/rpc_constants.textpb

```diff
@@ -0,0 +1,770 @@
+    unit: {
+      power10: 0
+      numerators: BYTES
+    }
+  }
+
+  measurement_descriptor {
+    name: "/rpc/client/response_bytes"
+    description: "Response bytes"
+    unit: {
+      power10: 0
+      numerators: BYTES
+    }
+  }
+
+  measurement_descriptor {
+    name: "/rpc/client/roundtrip_latency"
+    description: "RPC roundtrip latency msec"
+    unit: {
+      power10: -3
+      numerators: SECONDS
+    }
+  }
+
+  measurement_descriptor {
+    name: "/rpc/client/server_elapsed_time"
+    description: "Server elapsed time in msecs"
+    unit: {
+      power10: -3
+      numerators: SECONDS
+    }
+  }
+
+  measurement_descriptor {
+    name: "/rpc/client/uncompressed_request_bytes"
+    description: "Uncompressed Request bytes"
+    unit: {
+      power10: 0
+      numerators: BYTES
+    }
+  }
+
+  measurement_descriptor {
+    name: "/rpc/client/uncompressed_response_bytes"
+    description: "Uncompressed Response bytes"
+    unit: {
+      power10: 0
+      numerators: BYTES
+    }
+  }
+
+  #Census defined rpc server MeasurementDescriptors.
+  measurement_descriptor {
+    name: "/rpc/server/error_count"
+    description: "RPC Errors"
+    unit: {
+      power10: 0
+      numerators: BYTES
+    }
+  }
+
+  measurement_descriptor {
+    name: "/rpc/server/request_bytes"
+    description: "Request bytes"
+    unit: {
+      power10: 0
+      numerators: BYTES
+    }
+  }
+
+  measurement_descriptor {
+    name: "/rpc/server/response_bytes"
+    description: "Response bytes"
+    unit: {
+      power10: 0
+      numerators: BYTES
+    }
+  }
+
+  measurement_descriptor {
+    name: "/rpc/server/server_elapsed_time"
+    description: "Server elapsed time in msecs"
+    unit: {
+      power10: -3
+      numerators: SECONDS
+    }
+  }
+
+  measurement_descriptor {
+    name: "/rpc/server/server_latency"
+    description: "Latency in msecs"
+    unit: {
+      power10: -3
+      numerators: SECONDS
+    }
+  }
+
+  measurement_descriptor {
+    name: "/rpc/server/uncompressed_request_bytes"
+    description: "Uncompressed Request bytes"
+    unit: {
+      power10: 0
+      numerators: BYTES
+    }
+  }
+
+  measurement_descriptor {
+    name: "/rpc/server/uncompressed_response_bytes"
+    description: "Uncompressed Response bytes"
+    unit: {
+      power10: 0
+      numerators: BYTES
+    }
+  }
+
+  measurement_descriptor {
+    name: "/rpc/server/started_count"
+    description: "Number of RPCs started"
+    unit: {
+      power10: 0
+      numerators: SCALAR
+    }
+  }
+
+  measurement_descriptor {
+    name: "/rpc/server/finished_count"
+    description: "Number of RPCs finished"
+    unit: {
+      power10: 0
+      numerators: SCALAR
+    }
+  }
+
+  # Census defined rpc client ViewDescriptors.
+  view_descriptor {
+    name: "rpc client error_count"
+    description: "RPC Errors"
+    measurement_descriptor_name: "/rpc/client/error_count"
+    distribution_aggregation: {
+    }
+    tag_keys: "OpStatus"
```

Comment:
Can also use list syntax for tag keys, and add comments, maybe something like:

tag_keys: [
  'OpStatus',  # Tag value will reflect status of operation
  'method' # Tag value will be RPC service/method name.
]